rules/nette-kdyby/src/Naming/VariableNaming.php

```diff
@@ -243,13 +248,20 @@ private function resolveFromPropertyFetch(PropertyFetch $propertyFetch): string
         return $varName . ucfirst($propertyName);
     }
 
-    private function resolveFromMethodCall(MethodCall $methodCall): ?string
+    private function resolveFromMethodCall(Expr $expr): ?string
     {
-        if ($methodCall->name instanceof MethodCall) {
-            return $this->resolveFromMethodCall($methodCall->name);
+        if (! $expr instanceof MethodCall
+            && ! $expr instanceof NullsafeMethodCall
+            && ! $expr instanceof StaticCall
+        ) {
+            throw new ShouldNotHappenException('Only method calls are supported');
```

Comment:
Better mention the invalid node, so we can have narrow feedback.

Rough code (need test in real code and add `$allowedTypes` variable):

```suggestion
            $message = sprintf(
                 'Only "%s" are supported, "%s" given',
                 implode('", "', $allowedTypes),
                 get_class($expr)
            );
            throw new ShouldNotHappenException($message);
```

Reply:
Done. Btw

```bash
> grep -ir "ShouldNotHappenException();" . | wc -l
     154
> grep -ir "ShouldNotHappenException([^)]" . | wc -l
      51
```

Reply:
What does it mean

Reply:
There are 154 throws of `ShouldNotHappenException` without any message at all.

Versus 51 with some message